src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -474,6 +477,10 @@ export class CellModel extends Disposable implements ICellModel {
 			if (!kernel) {
 				return false;
 			}
+			// Add telemtry for the run cell
```

Comment:
Just remove this comment - it doesn't really provide anything that isn't clear from the code itself